python/src/aiconfig/editor/client/src/reducers/aiconfigReducer.ts

```diff
@@ -168,21 +168,23 @@ export default function aiconfigReducer(
     }
     case "UPDATE_PROMPT_MODEL": {
       return reduceReplacePrompt(dirtyState, action.id, (prompt) => {
-        // TODO: Consolidate settings based on schema union, server-side
-        // For now, just keep all the settings to match the server-side implementation
-        let modelSettings;
-        const promptModel = prompt.metadata?.model;
-        if (promptModel && typeof promptModel !== "string") {
-          modelSettings = promptModel.settings;
-        }
+        // TODO (rossdan): Uncomment below once we are able to handle updating model names for HF tasks or sub-classes
+        // // TODO: Consolidate settings based on schema union, server-side
+        // // For now, just keep all the settings to match the server-side implementation
+        // let modelSettings;
+        // const promptModel = prompt.metadata?.model;
+        // if (promptModel && typeof promptModel !== "string") {
+        //   modelSettings = promptModel.settings;
+        // }
         return {
           ...prompt,
           metadata: {
             ...prompt.metadata,
             model: action.modelName
               ? {
                   name: action.modelName,
-                  settings: modelSettings,
+                  // TODO (rossdanlm): For now just clearing the model settings whenever new model name is selected
```

Comment:
Wait, why doesn't this just remove 'model' if it exists instead? We were persisting he settings intentionally due to product feedback that changing the model from say gpt-4 to gpt-3.5 was dropping all of the settings even though they're supported

Reply:
Sure I can do that